src/cpu/static_inst.hh

```diff
@@ -307,6 +313,13 @@ class StaticInst : public RefCounted, public StaticInstFlags
         panic("buildRetPC not defined!");
     }
 
+    size_t size() const
+    {
+      if (_size == 0) panic("Instruction size not set!");
```

Comment:
This could use a more detailed error message. Maybe "Instruction size for this ISA is not set for branch predictor that needs instruction size. Either use a different branch predictor or this ISA needs to be extended with the instruction size."

Also, I think this should be a `fatal` as it is a configuration error, not a bug.